klogr/klogr.go

```diff
@@ -191,8 +209,8 @@ func pretty(value interface{}) string {
 	return strings.TrimSpace(string(buffer.Bytes()))
 }
 
-func (l klogger) Info(msg string, kvList ...interface{}) {
-	if l.Enabled() {
+func (l klogger) Info(level int, msg string, kvList ...interface{}) {
+	if l.Enabled(level) {
```

Comment:
you should not need to check Enabled().  logr will do that and will only call this if true.

Reply:
on it.